css/style.css

```diff
@@ -6,14 +6,551 @@ body {
+    }
+    .text-block p{
+        width: 50%;
+        font-size: 1.2rem;
+        margin: 0 auto;
+        margin-top: -1rem;
+        margin-bottom: 1rem;
+    }
+
+    .main-img{
+        width: 190%;
+    }
+
+
+    .burger-nav{
+        display: block;
+        position: absolute;
+       right: 4%;
+    }
+    
+    
+    
+    .topnav ul{
+        overflow: hidden;
+        background-color: rgb(255, 255, 255);
+        margin: auto;
+        padding: 0;
+        height:0;
+        top:0;
+        right: 0;
+        position: absolute;
+        width: 100%;
+            
+        }
+          
+        .topnav ul.open{
+            height: auto;
+        }
+          
+        .topnav ul li{
+            margin-top: 2rem;
+            display: block;
+            text-align: left;
+            display: flex;
+            flex-wrap: wrap;
+            flex-direction: column;
+        }
+          
+        .topnav ul li a{
+            color: rgb(47, 47, 47);
+            font-size: 1.5rem;
+            padding: 0.5rem;
+            border-bottom: 0.1rem solid rgb(182, 182, 182);
+            
+        }
+          
+          .topnav{
+              height: 0;
+              z-index: 1;
+          }
+
+/*phone "store" */    
+    
+    .connect{
+        align-items: center;
+        justify-content: center;
+    
+    }
+    
+    .connect img{
+        max-width: 100%;
+        width: 100%;
+    }
+    .connect_text{
+        margin: 2rem 0 2rem 0;
+        text-align: center;
+        margin-right: 0;
+    }
+    
+    .container_store{
+        grid-template-columns: 1fr;
+        gap: 2rem;
+        grid-template-rows: auto;
+        align-items: center;
+    }
+    
+    
+    .form{
+        margin: 0 auto;
+        padding: 0;
+        max-width: 100%;
+        align-content: center;   
+        border-radius: solid green;
+    }
+
+    .form h1{
+        padding: 0 3rem;
+    }
+
+    .name-container{
+        margin: 0 auto;
+        padding: 0 auto;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        width: auto;
+    }
+
+
+
+    .first-name{
+        margin: 0 auto;
+        text-align: left;
+        padding: 0;
+        height: auto;
+        max-height: 100%;
+        margin-bottom: 0.5rem;
+    }
+
+    #first-name, #last-name, #city, #postcode{
+        height: 2.5rem;
+        width:18rem;
+        border: solid rgb(121, 121, 121) 0.08rem;
+        border-radius: 0.2rem;
+        
+        border: solid;
+        
+    }
+
+    .last-name, .postcode{
+        margin-left: 0;
+    }
+
+    #adress{
+        height: 2.5rem;
+        width:18rem;
+        border: solid rgb(121, 121, 121) 0.08rem;
+    }
+
+    
+
+    #city{
+        height: 3rem;
+        width:18.3rem;
+        margin-bottom: 0.5rem;
+    }
+
+    .container_store img{
+        max-width: 100%;
+        width: auto;
+    }
+
+    .right-align{
+        padding: 0 3rem;
+    }
+    
 
+    
+    }
 
```

Comment:
i find it useful to write comments before each part of styling code, which explains what part of html is going to be styled or what is expected.